cerberus/forms.py

```diff
@@ -1,33 +1,10 @@
 # Django
 from django import forms
 
-# Third Party
-from djmoney.forms import MoneyWidget
-
 # Locals
 from .models import Booking, Charge, Customer, Invoice, Pet, Service, Vet
-
-
-class SingleMoneyWidget(MoneyWidget):
-    def __init__(self, attrs=None, *args, **kwargs):
-        if attrs is None:
-            attrs = {}
-        super().__init__(
-            amount_widget=forms.NumberInput(
-                attrs={
-                    **{
-                        "step": "any",
-                    },
-                    **attrs,
-                }
-            ),  # type: ignore
-            currency_widget=forms.HiddenInput(),
-            *args,
-            **kwargs,
-        )
-
-    def id_for_label(self, id_):
-        return f"{id_}_0"
+from .utils import minimize_whitespace
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Consider removing unused import.

The import 'minimize_whitespace' from '.utils' does not seem to be used in 'forms.py'. If it's not needed, consider removing it to keep the code clean.